config/dev.webpack.config.js

```diff
@@ -1,13 +1,23 @@
-
 const { resolve } = require('path');
 const config = require('@redhat-cloud-services/frontend-components-config');
 const { config: webpackConfig, plugins } = config({
     rootFolder: resolve(__dirname, '../'),
     debug: true,
-    https: false,
     port: 8002,
+    useFileHash: false,
 });
 
+plugins.push(
+    require('@redhat-cloud-services/frontend-components-config/federated-modules')({
+        root: resolve(__dirname, '../'),
+        useFileHash: false,
+        moduleName: 'image_builder',
+        exposes: {
+            './RootApp': resolve(__dirname, '../src/bootstrap-dev'),
```

Comment:
This one should point to `AppEntry.js` otherwise you'll end with 2 apps in virtual DOM.